docs/marketplace.vs-building-blocks.md

```diff
@@ -24,18 +22,18 @@ The table below describes each capability and to what extent it is supported by
 |                                           | Building Blocks                                                                   | Service Brokers                                                                                                                       |
 |-------------------------------------------|-----------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
 | Provision resources with Terraform        | ✅ This works very easily                                                          | 🟨 Works using [UniPipe Service Broker](https://github.com/meshcloud/unipipe-service-broker), but requires more implementation effort |
-| Run Custom Processes outside of Terraform | ⏰                                                                                 | ✅ Service Brokers run on OSB API which means you can build anything you want as long as the interface is compliant with OSB API       |
+| Run Custom Processes outside of Terraform | ✅ Async provisioning is supported and allows writing back status updates from external systems into meshStack                                                                                 | ✅ Service Brokers run on OSB API which means you can build anything you want as long as the interface is compliant with OSB API       |
```

Comment:
Shouldn't we rather mention that you can trigger external processes using Terraform? Like committing a file into a CI/CD repo

Reply:
That the triggering is to be done with terraform is correct. The important information from my perspective is not the specific form of the glue you need, but that you can do async provisioning via tools you control and that this isn't an afterthought but something we designed meshStack for